src/bundles/ApplicationsAdmin/components/AgencyList/AgencyList.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { loadAgencies } from '../../redux/modules/agency'
+
+class AgencyList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      loading: false,
+      agencies: [],
+      saved: null
+    }
+
+  }
+
+  componentDidMount() {
+    this.setState({
+      loading: true
+    })
+    this.props.loadAgencies()
+      .then(r => {
+        if (r.status === 200) {
+          return r.json()
```

Comment:
Should this return `Promise.resolve(r.json())`? Or perhaps just set the `agencies` state here and avoid having the extra `.then()`?

Reply:
r.json() returns a promise...